dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/BaseClasses_INIT.mo

```diff
@@ -1,20 +1,84 @@
 within Dynawo.Electrical.HVDC;
 
 /*
-* Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+* Copyright (c) 2023, RTE (http://www.rte-france.com)
 * See AUTHORS.txt
 * All rights reserved.
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, you can obtain one at http://mozilla.org/MPL/2.0/.
 * SPDX-License-Identifier: MPL-2.0
 *
-* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
 */
 
 package BaseClasses_INIT
   extends Icons.BasesPackage;
 
+partial model BasePV_INIT "Base initialization model for PV HVDC"
+
+  parameter Types.ActivePowerPu P1RefSetPu "Start value of active power reference at terminal 1 in pu (base SnRef) (receptor convention)";
+  parameter Types.ReactivePower Q1Nom "Nominal reactive power in Mvar at terminal 1";
+
+  Types.VoltageModulePu U1Ref0PuVar "Start value of the voltage regulation set point in pu (base UNom) at terminal 1";
+  Types.ReactivePowerPu QInj10PuQNom "Reactive power at terminal 1 in pu (base Q1Nom) (generator convention)";
```

Comment:
Swap the lines (alphabetical order) and move them after U1Ref0Pu (an initial parameter).